src/main/java/com/exasol/sql/rendering/ColumnsDefinitionRenderer.java

```diff
@@ -0,0 +1,122 @@
+package com.exasol.sql.rendering;
+
+import com.exasol.datatype.type.*;
+import com.exasol.datatype.type.Boolean;
+import com.exasol.sql.*;
+
+/**
+ * Renderer for columns definition.
+ */
+public class ColumnsDefinitionRenderer extends AbstractFragmentRenderer implements ColumnDefinitionVisitor {
+    /**
+     * Create a new instance of an {@link AbstractFragmentRenderer}-based class.
+     *
+     * @param config renderer configuration
+     */
+    public ColumnsDefinitionRenderer(final StringRendererConfig config) {
+        super(config);
+    }
+
+    @Override
+    public void visit(final ColumnsDefinition columnsDefinition) {
+        if (!columnsDefinition.getColumns().isEmpty()) {
+            append(" (");
```

Comment:
Having a leading space here is not ideal. The columns definition is the part in the brackets. If a renderer needs a space between whatever comes first and the columns definition it should be responsible for adding that itself.

Reply:
I will use   appendSpace();